internal/controller/clusterorder_webhook.go

```diff
@@ -0,0 +1,45 @@
+package controller
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
+
+	cloudkitv1alpha1 "github.com/innabox/cloudkit-operator/api/v1alpha1"
+)
+
+func triggerWebHook(ctx context.Context, url string, instance *cloudkitv1alpha1.ClusterOrder) error {
+	log := ctrllog.FromContext(ctx)
+	log.Info("Triggering webhook " + url)
+
+	jsonData, err := json.Marshal(instance)
```

Comment:
Fine for now perhaps, but I think we'll want to give this a minimal envelope so we can pass other stuff in addition to the cluster order. For example, maybe we'd pass a token or other resources with useful context. Up to you if you want to establish that now or cross that bridge when it becomes useful.

Reply:
I think I'll wait until we have a concrete requirement. @adriengentil had some thoughts about tokens and impersonation that may mean we don't need to worry about passing tokens around.

Reply:
As you said, I don't think we need it immediately. Another advantage of the envelope would be to decouple ourselves from HTTP webhooks as we rely on the URL path to route the action (create/delete), it would aloow us to be able to use other communication ways (kafka for example).